### ISSUE TITLE
it seems this isn't working on iOS 18

### ISSUE THREAD

**Issue by kelvinOfoli** [LKHcoding/react-native-blur-effect#4]
![image](https://github.com/user-attachments/assets/946c0051-f280-4fb3-a307-91697a21f20c)

The items in the background are supposed to be blurred out.. works perfect on Android and iOS versions below 18...

**Comment by kelvinOfoli** [comment]
I found this,
https://discussions.apple.com/thread/255764118?sortBy=rank

**Comment by vstacked** [comment]
did u find another solution or an alternative @kelvinOfoli ?

**Comment by kelvinOfoli** [comment]
@vstacked 
I resulted to using "@react-native-community/blur" just for iOS 18 and above

```
  if (majorVersionIOS >= 18 && isIOS) {
    return <RNBlurView style={styles.container} blurType={blurType} />;
  }
```

**Comment by vstacked** [comment]
Yes, in the end I also used that, but with that library I encountered another problem, which was a white screen. According to this https://github.com/software-mansion/react-native-screens/issues/1882#issuecomment-2435173230, need to set animation to none. Did you @kelvinOfoli experience something similar?

**Comment by kelvinOfoli** [comment]
I think I remember experiencing something like this only for android..
from what I remember giving the container of the blueView (@react-native-community/blur)  a 
` overflow: "hidden" `
 fixed the issue.

**Comment by vstacked** [comment]
> I think I remember experiencing something like this only for android.. from what I remember giving the container of the blueView (@react-native-community/blur) a `overflow: "hidden"` fixed the issue.

Oh I see, okay I'll try it later, thank you
